app/[locale]/director-message/page.tsx

```diff
@@ -10,55 +71,212 @@ export default async function DirectorCorner({
   const text = (await getTranslations(locale)).DirectorMessage;
   return (
     <>
-      <section
-        className="container"
+      {/* For DIRECTOR CARD */}
+      <div
+        className="w-full bg-cover bg-center"
         style={{
-          backgroundImage: `linear-gradient(rgba(249, 245, 235, 1) 0%, rgba(249, 245, 235, 0.85) 85%, rgba(249, 245, 235, 1) 100%), url('${getS3Url()}/assets/temple-1.jpeg')`,
+          backgroundImage: `url('${getS3Url()}/assets/director-bg.jpeg')`, // Use a dedicated director section bg
         }}
       >
-        <Heading
-          text={text.title.toUpperCase()}
-          heading="h3"
-          glyphDirection="dual"
-        />
-        <article className="flex max-md:flex-col">
-          <p className="text-lg  max-md:rounded-t md:w-full md:rounded-r ">
-            {text.message.slice(0, 7).map((message, index) => (
-              <span key={index} className="mb-5 block">
-                {message}
-              </span>
-            ))}
-          </p>
-        </article>
-        <article className="mx-auto flex items-center justify-center max-md:flex-col">
-          <p className="w-fit max-w-4xl text-center text-lg max-md:rounded-t md:rounded-r">
-            {text.message.slice(7, 9).map((message, index) => (
-              <span key={index} className="mb-5 block">
-                {message}
-              </span>
-            ))}
-          </p>
-        </article>
-        <article className="flex max-md:flex-col">
-          <p className="text-lg  max-md:rounded-t md:w-full md:rounded-r ">
-            {text.message.slice(9, 11).map((message, index) => (
-              <span key={index} className="mb-3 block">
-                {message}
-              </span>
+        <section className="container mx-auto">
+          <Heading
+            text={directorCard.title.toUpperCase()}
+            heading="h2"
+            glyphDirection="dual"
+          />
+          {/* DIRECTOR CARD */}
+          <article className="flex h-fit w-full items-center justify-center">
+            <Card
+              imageAlt={directorCard.alt}
+              imageSrc={directorCard.image}
+              name={directorCard.name}
+              designation={`${directorCard.position}, ${directorCard.institute}`}
+              contactDetails={[
+                {
+                  label: Array.isArray(directorCard.contact.phone)
+                    ? directorCard.contact.phone.join(' ')
+                    : directorCard.contact.phone,
```

Comment:
`directorCard` is statically defined above in this file, which means we can guarantee the type of `directorCard.contact.phone`; so this check is redundant, it will always hit one of the outcomes, and never the other.

Same with the checks below.